activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb

```diff
@@ -497,8 +497,11 @@ def copy_table(from, to, options = {})
                  options[:rename][column.name.to_sym] ||
                  column.name) : column.name
 
+              type = lookup_cast_type_from_column(column)
+              default = type.deserialize(column.default) if column.has_default?
```

Comment:
Small nit: does it make sense to avoid looking up the cast type if there's no default?
```suggestion
              default = if column.has_default?
                type = lookup_cast_type_from_column(column)
                type.deserialize(column.default)
              end
```

Reply:
It does not hurt if we check, but it may be cleaner (clearer) to do a check.
Thanks for the suggestion 👍